tasks/December/tasks.js

```diff
@@ -0,0 +1,153 @@
+// Napisz funkcję isAdult(age), która przyjmuje wiek jako argument i zwraca true,// jeśli użytkownik ma 18 lat lub więcej, a false w przeciwnym razie.
+const isAdult = (age) => {
+	return age >= 18 ? true : false;
```

Comment:
Wystarczy samo `return age >= 18`. Ponieważ to zwróci true albo false, więc nie musimy dodatkowo tego pisać.